src/api.v2/model/SampleFile.js

```diff
@@ -0,0 +1,33 @@
+const BasicModel = require('./BasicModel');
+const sqlClient = require('../../sql/sqlClient');
+
+const tableNames = require('./tableNames');
+
+const sampleFields = [
+  'id',
+  'sample_file_type',
+  'size',
+  's3_path',
```

Comment:
Why aren't these fields part of `Sample`? Are they going to be different for sample files that are from the same sample?

```  
'sample_file_type',
  'size',
  's3_path',
```